src/Microsoft.SqlTools.ServiceLayer/Admin/Database/DatabasePrototype.cs

```diff
@@ -730,6 +744,8 @@ public bool HasSameValueAs(DatabaseData other)
             {
                 bool result =
                     (this.recoveryModel == other.recoveryModel) &&
+                    (DateTime.Compare(this.lastBackupDate, other.lastBackupDate) == 0) &&
+                    (DateTime.Compare(this.lastLogBackupDate, other.lastLogBackupDate) == 0) &&
```

Comment:
Also, all these parens are extraneous, but I can see the desire to stick with the existing pattern